### PR TITLE
[feature](filecache) add a const parameter to control the cache version

### PR DESCRIPTION
**Pull request by AshinGau**
# Proposed changes

Add a const parameter to control the cache version:
```
IFileCache::USE_FIE_VERSION2 = true;
```

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by AshinGau** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.79 seconds
 stream load tsv:          468 seconds loaded 74807831229 Bytes, about 152 MB/s
 stream load json:         37 seconds loaded 2358488459 Bytes, about 60 MB/s
 stream load orc:          75 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230306023300_clickbench_pr_108939.html

File: be/src/io/cache/block/block_file_cache.h [hunk 1]
```diff
@@ -43,8 +43,8 @@ class IFileCache {
     friend struct FileBlocksHolder;
 
 public:
-    static const std::string FILE_CACHE_VERSION;
-    static const int KEY_PREFIX_LENGTH;
+    static constexpr bool USE_FIE_VERSION2 = true;
```
Comment (morningman) [inline be/src/io/cache/block/block_file_cache.h:46]: Add commend to explain this field

**Comment by AshinGau** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**